patches/react-native+0.66.4.patch

```diff
@@ -1,3 +1,34 @@
+diff --git a/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.js b/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.js
+index e497288..696781f 100644
+--- a/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.js
++++ b/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.js
+@@ -1178,11 +1178,6 @@ class ScrollView extends React.Component<Props, State> {
+         );
+       }
+     }
+-    if (Platform.OS === 'android') {
+-      if (this.props.keyboardDismissMode === 'on-drag' && this._isTouching) {
+-        dismissKeyboard();
+-      }
+-    }
+     this._observedScrollSinceBecomingResponder = true;
+     this.props.onScroll && this.props.onScroll(e);
+   };
+@@ -1299,6 +1294,14 @@ class ScrollView extends React.Component<Props, State> {
+    */
+   _handleScrollBeginDrag: (e: ScrollEvent) => void = (e: ScrollEvent) => {
+     FrameRateLogger.beginScroll(); // TODO: track all scrolls after implementing onScrollEndAnimation
++
++    if (
```

Comment:
nit: Would be nice to add a comment in the patch about where this comes from (link to the RN commit).